metrics_sdk/lib/opentelemetry/sdk/metrics/state/metric_stream.rb

```diff
@@ -30,30 +30,60 @@ def initialize(
             @instrument_kind = instrument_kind
```

Comment:
I assume that initializing the MetricStream falls into this part of the spec: https://opentelemetry.io/docs/specs/otel/metrics/sdk/#stream-configuration 

If so, we might want to consider updating the defaults of the MetricStream to more closely match the spec. I don't think we need to do this as part of this PR. If we don't, we may want to create an issue to tackle this later on.

Reply:
Yeah, it's probably best to address it in different PR dedicates for metric_stream.

View seems doesn't need to have `attribute_keys`, but I was confused by this [spec's example](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#view)
```
meter_provider
    .add_view("X", aggregation=SumAggregation())
    .add_view("*", attribute_keys=["a", "b"]) # wildcard view matches everything, including X
    ...
```

Reply:
Hmm... that is odd. I'm on board for leaving this as-is for this PR. New issue to track the work: https://github.com/open-telemetry/opentelemetry-ruby/issues/1656